### ISSUE TITLE
镜像路径出现两个＂/＂ push 镜像会导致UI Crash

### ISSUE THREAD

**Issue by huxos** [vmware/harbor#1678]
新建项目 test
打镜像标签 reg.example.com/test/test/nginx:latest 
然后push 这个镜像导致UI崩溃，并且列表里面不会展示这个镜像
ui 的日志如下：

```
2017-03-20T10:12:21Z [DEBUG] [notification.go:107]: receive an event: 
----ID: 1d109575-02ce-4269-9e25-e3ea039dc6c1 
----target: test/test/nginx: 
----digest: sha256:6b914bbcb89e49851990e064568dceee4d53a462f316ec36207599c12ae9ba65 
----action: push 
----mediatype: application/octet-stream 
----user-agent: docker/17.03.0-ce go/go1.8 git-commit/60ccb2265b kernel/4.9.11-1-ARCH os/linux arch/amd64 UpstreamClient(Docker-Client/17.03.0-ce \(linux\))
2017/03/20 10:12:21 [router.go:829][D] | POST       | /service/notifications                   | 448.18µs         | match      | /service/notifications                   | 
2017-03-20T10:12:21Z [DEBUG] [notification.go:107]: receive an event: 
----ID: 18bf0b16-7fef-44d8-9b75-2c895962eb86 
----target: test/test/nginx: 
----digest: sha256:6b914bbcb89e49851990e064568dceee4d53a462f316ec36207599c12ae9ba65 
----action: pull 
----mediatype: application/octet-stream 
----user-agent: docker/17.03.0-ce go/go1.8 git-commit/60ccb2265b kernel/4.9.11-1-ARCH os/linux arch/amd64 UpstreamClient(Docker-Client/17.03.0-ce \(linux\))
2017/03/20 10:12:21 [router.go:829][D] | POST       | /service/notifications                   | 594.529µs        | match      | /service/notifications                   | 
2017-03-20T10:12:21Z [DEBUG] [notification.go:107]: receive an event: 
----ID: c7d2eaa1-3087-46c5-bac6-80e0ece2dbf8 
----target: test/test/nginx:latest 
----digest: sha256:7f9c22c6b2ba14c609588a794fb40366c5d1eea2c70b8bb7474583994dab9e70 
----action: push 
----mediatype: application/vnd.docker.distribution.manifest.v2+json 
----user-agent: docker/17.03.0-ce go/go1.8 git-commit/60ccb2265b kernel/4.9.11-1-ARCH os/linux arch/amd64 UpstreamClient(Docker-Client/17.03.0-ce \(linux\))
2017-03-20T10:12:21Z [DEBUG] [notification.go:123]: add event to collect: c7d2eaa1-3087-46c5-bac6-80e0ece2dbf8
2017/03/20 10:12:21 [router.go:829][D] | POST       | /service/notifications                   | 855.006µs        | match      | /service/notifications                   | 
2017-03-20T10:12:21Z [ERROR] [accesslog.go:154]: error in AccessLog: Error 1048: Column 'project_id' cannot be null 
2017-03-20T10:12:21Z [ERROR] [notification.go:71]: failed to add access log: Error 1048: Column 'project_id' cannot be null
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x0 pc=0x51ae75]

goroutine 240 [running]:
panic(0xc22340, 0xc82000a0e0)
	/usr/local/go/src/runtime/panic.go:481 +0x3e6
github.com/vmware/harbor/src/ui/api.GetPoliciesByRepository(0xc820344c20, 0xf, 0x0, 0x0, 0x0, 0x0, 0x0)
	/go/src/github.com/vmware/harbor/src/ui/api/utils.go:161 +0x135
github.com/vmware/harbor/src/ui/api.TriggerReplicationByRepository(0xc820344c20, 0xf, 0xc820344f30, 0x1, 0x1, 0xd42140, 0x8)
	/go/src/github.com/vmware/harbor/src/ui/api/utils.go:171 +0x50
created by github.com/vmware/harbor/src/ui/service.(*NotificationHandler).Post
	/go/src/github.com/vmware/harbor/src/ui/service/notification.go:89 +0x603
```

**Comment by reasonerjt** [comment]
I think it's been fixed?
@ywk253100

**Comment by ywk253100** [comment]
Yes. It has been fixed in #1325 .
